src/CMakeLists.txt

```diff
@@ -86,6 +86,8 @@ add_files(
     bitmap_type.h
     bmp.cpp
     bmp.h
+    bootstrap.cpp
+    bootstrap_emscripten_gui.cpp
```

Comment:
Would it be better to make this conditional compile in cmake, rather than ifdef in the cpp file?